src/neo/Network/P2P/Payloads/TransactionVersion.cs

```diff
@@ -0,0 +1,9 @@
+namespace Neo.Network.P2P.Payloads
+{
+    public enum TransactionVersion : byte
```

Comment:
Should this be a flag? Because we may have more features in the future.

Reply:
if it's a flag, You can send Normal, and Oracle Request together.

Reply:
I think it's okay to be a flag. It is the same as attributes.

Reply:
Please, check https://github.com/neo-project/neo/pull/1609